docs/reference/commandline/dockerd.md

```diff
@@ -594,6 +594,11 @@ The Docker daemon relies on a
 (invoked via the `containerd` daemon) as its interface to the Linux
 kernel `namespaces`, `cgroups`, and `SELinux`.
 
+The Docker daemon will automatically start a `containerd` daemon, unless you
```

Comment:
`--containerd` is a flag not an option. Suggested edit :

By default, the Docker daemon automatically starts `containerd`. If you want to manually control `containerd` startup, pass the path to the `containerd` socket using the `--containerd` flag. For example:

```bash
dockerd /var/run/dev/docker-containerd.sock
```